tool/tctl/common/tctl.go

```diff
@@ -112,6 +116,9 @@ func Run(commands []CLICommand, loadConfigExt LoadConfigFn) {
 		StringVar(&ccf.IdentityFilePath)
 	app.Flag("insecure", "When specifying a proxy address in --auth-server, do not verify its TLS certificate. Danger: any data you send can be intercepted or modified by an attacker.").
 		BoolVar(&ccf.Insecure)
+	app.Flag("ignore-config", fmt.Sprintf("Ignore Teleport config [%v] if present on disk.", defaults.ConfigFilePath)).
+		Short('z').
```

Comment:
`-i` and `-c` were taken so I wasn't sure what to use; `-z` seemed to be innocuous and unlikely to be used for anything else. I wanted a short flag to avoid having to type `--ignore-config` all the time.